scanpipe/models.py

```diff
@@ -3870,36 +3934,135 @@ def get_payload(self, pipeline_run):
             "run": run_serializer.data,
         }
 
+        if self.include_summary:
+            summary_file = self.project.get_latest_output(filename="summary")
+            summary = json.loads(summary_file.read_text()) if summary_file else {}
+            payload["summary"] = summary
+
+        if self.include_results:
+            results_file = to_json(self.project)
+            payload["results"] = json.loads(results_file.read_text())
+
         return payload
 
-    def deliver(self, pipeline_run):
+    def deliver(self, pipeline_run, timeout=10):
         """Deliver this Webhook by sending a POST request to the `target_url`."""
+        logger.info(f"Delivering Webhook {self.uuid}")
+
+        if not self.is_active:
+            logger.info(f"Webhook {self.uuid} is not active.")
+            return False
+
         payload = self.get_payload(pipeline_run)
+        delivery = WebhookDelivery(
+            project=self.project,
+            webhook_subscription=self,
+            run=pipeline_run,
+            target_url=self.target_url,
+            payload=payload,
+        )
 
-        logger.info(f"Sending Webhook uuid={self.uuid}.")
         try:
             response = requests.post(
```

Comment:
I am having trouble sending the project results back to purldb because the authentication token is not used here.